### PR TITLE
Changed S1SingleScatter Cut to use the correct alternate interaction

### PR DESCRIPTION
**Pull request by jpienaar13**

### CODE REVIEW

File: lax/lichens/sciencerun0.py [hunk 1]
```diff
@@ -211,12 +211,12 @@ class S1SingleScatter(Lichen):
     Contact: Jacques <jpienaa@purdue.edu>
     """
 
-    version = 0
+    version = 0.1
```
Comment (tunnell) [inline lax/lichens/sciencerun0.py]: Version integer, make 2.

**Review by tunnell — DISMISSED** [review]
Can you fix tabs spaces issue that was flagged?

**Review by tunnell — CHANGES_REQUESTED** [review]
need to fix version, also non-line-lone pep8

File: lax/lichens/sciencerun0.py (hunk 1 above)
Comment (pdeperio) [reply lax/lichens/sciencerun0.py]: (comment to block merging)

**Review by pdeperio — CHANGES_REQUESTED** [review]

**Review by pdeperio — APPROVED** [review]

**Review by tunnell — APPROVED** [review]

**Pull request merged**